Makefile

```diff
@@ -441,6 +441,7 @@ release-minikube: out/minikube checksum
 
 out/docker-machine-driver-kvm2: pkg/minikube/translate/translations.go
 ifeq ($(MINIKUBE_BUILD_IN_DOCKER),y)
+	docker inspect -f '{{.Id}} {{.RepoTags}}' $(KVM_BUILD_IMAGE) || \
```

Comment:
The`docker inspect` won't be necessary after we have this image on the registry, correct? Because `docker pull` will only pull if the image doesn't exist. Maybe we can add a `TODO: remove this when ....`

Reply:
It will pull it every single time (maybe because we're using "latest") but anyway.

```
Using default tag: latest
Error response from daemon: manifest for gcr.io/k8s-minikube/kvm-build-image:latest not found
```

Reply:
hmm, for example, if I try to `docker pull nginx` which is on my machine already I get:
```
Using default tag: latest
latest: Pulling from library/nginx
Digest: sha256:eb3320e2f9ca409b7c0aa71aea3cf7ce7d018f03a372564dbdb023646958770b
Status: Image is up to date for nginx:latest
docker.io/library/nginx:latest
```

Reply:
It might be possible to use `docker images` instead of `docker inspect`, not sure ?

No `docker image exists`, unfortunately. (like there is with `podman image exists`)

Reply:
if you think worth double-checking before doing the pull, I'm cool with it, just that locally at least the second pull is really fast too, it probably just check the digest instead of doing a big download, but definitely not faster than checking with `docker inspect`

Reply:
The problem is if you build an image locally that _doesn't exist_ on the docker hub.
Then it will fail to pull, and thus fail to recognize that it exists in the local storage..

Reply:
You are right of course in that it won't actually have to pull any layers or anything.